kv.go

```diff
@@ -980,3 +965,73 @@ func (kv *kvs) Status() (KeyValueStatus, error) {
 
 	return &KeyValueBucketStatus{nfo: nfo, bucket: kv.name}, nil
 }
+
+// KeyValueStoreNames is used to retrieve a list of key value store names
+func (js *js) KeyValueStoreNames() <-chan string {
+	ch := make(chan string)
+	ctx, cancel := context.WithTimeout(context.Background(), defaultRequestWait)
```

Comment:
maybe derive this timeout from the ones that are part of the js context? 
https://github.com/nats-io/nats.go/blob/715a5917c806d2858f6fc80663fbf0910ad58ee3/js.go#L234-L238

Reply:
Looking at this, it seems I can drop creating the context altogether, as `l.Next()` has its own context (either from `js.opts.ctx` or from `js.opts.wait`). I changed it, take a look if it looks all right.